CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # CHANGELOG
 
 ## vNext
```

Comment:
Please, specify the version

Reply:
What do you think if we collect all the changes and create a release `0.19.2` in `support/0.19.x` ? 
 It might not be necessary to create a release for each individual change in `support` branch